src/libcmis/ws-object-type.cxx

```diff
@@ -66,7 +66,7 @@ WSObjectType& WSObjectType::operator=( const WSObjectType& copy )
 void WSObjectType::refresh( )
 {
     libcmis::ObjectTypePtr type = m_session->getType( m_id );
-    WSObjectType* const other = dynamic_cast< WSObjectType* >( type.get( ) );
+    const WSObjectType* const other = dynamic_cast< WSObjectType* >( type.get( ) );
```

Comment:
The type in the `dynamic_cast` should be updated too. (Even better: change just the type in the cast and use `const auto` for the variable decl.)

Reply:
Patch submitted here:
https://github.com/tdf/libcmis/pull/65